### ISSUE TITLE
Adaugat participanti la evenimente pe hub.rosedu.org

### ISSUE THREAD

**Issue by mariuscoto** [rosedu/meta#4]
TODO:
- [x] adaugat numele participantilor la ultimele evenimente ROSEdu: CDL si SW
  - CDL: http://hub.rosedu.org/activities/CDL/2016
  - SW: http://hub.rosedu.org/activities/SummerWorkshops/2016

**Comment by maria** [comment]
Not really. there are more workshops, the one you added is only the scala one.
